packages/wrangler/src/index.tsx

```diff
@@ -1311,7 +1313,7 @@ export async function main(argv: string[]): Promise<void> {
           legacyEnv={
             (args["legacy-env"] as boolean | undefined) ??
             config.legacy_env ??
-            false
+            LEGACY_ENV_DEFAULT
```

Comment:
oh hmm, this should just be replaced by `isLegacyEnv()`. no biggie.

Reply:
Fixed in my PR (https://github.com/cloudflare/wrangler2/pull/529/files#diff-5e47c8c5ec00bfdc010f06653422c5db6be55b5c61226212cf593b8c57098b6cR1145)